lib/ecmascript.ts

```diff
@@ -5234,12 +5262,16 @@ export function AddDurationToOrSubtractDurationFromPlainYearMonth(
 
   const calendar = GetSlot(yearMonth, CALENDAR);
   const fieldNames = CalendarFields(calendar, ['monthCode', 'year'] as const);
-  const fields = ToTemporalYearMonthFields(yearMonth, fieldNames);
+  const fields = ToTemporalYearMonthFields(yearMonth, fieldNames) as Required<Temporal.PlainYearMonthLike> & {
```

Comment:
Same issue here with this cast as above.

Reply:
Done - same as above.